### ISSUE TITLE
Update MyBB Installer Theme

### ISSUE THREAD

**Issue by kawaii** [mybb/mybb#3712]
As discussed in the Discord a few days ago between @Ben-MyBB and @justinsoltesz - we should bring the MyBB installer/upgrader into 2019 too with a slight makeover. There are relatively few components in the installer so this should be a nice easy one for us to get done while @euantorano works on the version rebases.

**Comment by ks129** [comment]
For one note: Please remove database tables list showing in installer. I don't want to scroll down and there is next button.

**Comment by euantorano** [comment]
Good point. If we do need that list for some reason, it should be in a fixed size box of its own with a scroll bar.

**Comment by Ben-MyBB** [comment]
Can https://github.com/mybb/mybb/issues/3620 be incorporated into this?

**Comment by euantorano** [comment]
I'd prefer to see 2 separate PRs for this and #3620 if possible, as it makes it easier to track through the git log.
